packages/core/src/Services/socket-general.js

```diff
@@ -100,6 +100,28 @@ const BinarySocketGeneral = (() => {
         const error_code = ObjectUtils.getPropertyValue(response, ['error', 'code']);
         switch (error_code) {
             case 'WrongResponse':
+                // TODO: Remove condition checks below for WrongResponse once mt5 is more reliable
```

Comment:
we also hit this on `transfer_between_accounts`, should we handle that too?

Reply:
That could be done, but I don't think it should be automated here to retry the same transfer again in the event of `WrongResponse` for that call. 

Maybe catch the error response from `transfer_between_accounts` page component (where the call is made), display a message there and ask user to try again?

Reply:
I think that's already done.